snprc_scheduler/src/org/labkey/snprc_scheduler/services/SNPRC_schedulerServiceValidator.java

```diff
@@ -113,7 +113,7 @@ public static void validateNewTimeline(Timeline timeline, Container c, User u, B
         {
             // All New Timeline, check json draft state (and other such checks here)
             //
-            if (timeline.getQcState() != null && !timeline.getQcState().equals(QCStateEnum.IN_PROGRESS.getValue()))
+            if (timeline.getQcState() != null && !timeline.getQcState().equals(QCStateEnum.getValueByName(c,u, QCStateEnum.IN_PROGRESS)))
```

Comment:
getValueByName() returns an Integer as per the implementation, does timeline.getQcState() return an Interger or String?

Reply:
Timeline stores QcState as an integer:     private Integer _qcState;